### PR TITLE
Add notes

### PR DESCRIPTION
**Pull request by omjadhav85**
Features added:
1. User now can add and edit notes
2. Added Loader and Toasts
3. Some minor fixes

**Note: Please only review the last commits**

https://github.com/bLAckBLAZER/my-notes-app/commit/bd47fb8334069711dd0c958eef126cdb75728e21

and 

https://github.com/bLAckBLAZER/my-notes-app/commit/a7ebb04d9c6df8389b82441684dee70c188a092d

This PR also contains changes of previous PRs.

### CODE REVIEW

File: src/utils/noteServerCalls.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import axios from "axios";
+import { defaultNewNoteData } from "../components/Note/defaultNewNoteData";
+// import { isPresentInList } from "./helperFunctions";
```
Comment (AnuragSahu11) [inline src/utils/noteServerCalls.js:2]: comments should be removed

**Review by AnuragSahu11 — APPROVED** [review]
Loader animation is nice, functionality working as expected, ready to merge.

File: src/utils/noteServerCalls.js (hunk 1 above)
Comment (omjadhav85) [reply src/utils/noteServerCalls.js:2]: Will surely do that in next PRs. Thanks!

**Review by junaidshaikh-js — APPROVED** [review]
Good to merge.

**Comment by omjadhav85** [comment]
Thanks Junaid and Anurag for the reviews. Merging after two reviews.

**Pull request merged**